src/core/server/integration_tests/saved_objects/migrations/group2/multi_node_split.test.ts

```diff
@@ -0,0 +1,292 @@
+/*
```

Comment:
nit: this one might need its own group, as starting multiple Kibanas, with delays increase the execution time, and the chances of failing due to timeouts (and retrying the whole CI job).

Reply:
It already has its own group, there aren't any other tests in `group2` anymore 😛 .
I will increase jest timeout though, thanks for raising the poing